about.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About-The Tech Guru</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose-pro" rel="stylesheet" />
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+
+  <body>
+    <div class="status-bar">
+      <ul>
+        <li>
+          <a href="https://www.facebook.com"
+            ><i class="fa-brands fa-facebook"></i
+          ></a>
+        </li>
+        <li>
+          <a href="https://www.twitter.com"
+            ><i class="fa-brands fa-twitter"></i
+          ></a>
+        </li>
+        <li><a href="#">Logout</a></li>
+        <li><a href="#">Mypage</a></li>
+        <li><a href="#">English</a></li>
+      </ul>
+    </div>
+    <!-- Header Section Start -->
+    <header class="about-img" id="header">
+      <nav>
+        <a class="logo" href="./index.html"
+          ><img src="./images/TechGuru-Logo.png" alt=""
+        /></a>
+        <i class="fa fa-bars menu-icon" id="menu-icon"></i>
+        <i class="fa fa-xmark x-icon" id="x-icon"></i>
+        <ul class="menu-items-desktop" id="menu-items-desktop">
+          <li id="home"><a href="./index.html">Home</a></li>
+          <li id="contact"><a href="#header">Contact</a></li>
+          <li id="developers"><a href="#partner">Partners</a></li>
+          <li><a class="btn-menu" href="">Tech Guru</a></li>
+        </ul>
+      </nav>
+      <ul class="menu-items" id="menu-items">
+        <li id="menu-home"><a href="./index.html">Home</a></li>
+        <li id="menu-about"><a href="./about.html">About</a></li>
+        <li id="menu-contact"><a href="#header">Contact Me</a></li>
+        <li id="menu-developers"><a href="#partner">Partners</a></li>
+      </ul>
+
+      <!-- Main header / Landing Page Content Section -->
```

Comment:
- [ ] The main section of the About page doesn't have the same design pattern as the given design. Please make sure you follow the design because as a developer you would need to stick to the design provided by the team. 

**Original Design**

![image](https://user-images.githubusercontent.com/58504655/181949319-0d7fcd62-220e-4580-b988-adc29657aba2.png)

**Cloned Design**

![image](https://user-images.githubusercontent.com/58504655/181949743-cf004343-bb50-456a-ae01-c866383f5444.png)